### ISSUE TITLE
When delete first tag from many tags in test suite, ride is freeze and force to close

### ISSUE THREAD

**Issue by princeaok** [robotframework/RIDE#2371]
**Test Steps**
1. I have four tags
2. I delete first tag
3. Ride freezes
4. Ride is closed

![image](https://user-images.githubusercontent.com/37319103/111954449-b52d7d80-8b1a-11eb-81cb-68743b06fe20.png)

**Remark**
Doesn't matter how big your project is, it will be crashed. (Test in just one test suite in folder and it can be reproduced as well.)

**Comment by HelioGuilherme66** [comment]
@princeaok I assume you are using Windows operating system, but need to know more information. Please paste here the content of RIDE->Tools->View RIDE Log.
Here is mine (development version), where I could not reproduce your problem:
```
20210322 09:25:29.238 [INFO]: Found Robot Framework version 4.0 from C:\Users\Username\AppData\Roaming\Python\Python39\site-packages\robot.

20210322 09:25:29.239 [INFO]: Started RIDE v2.0b2.dev2 using python version 3.9.1 with wx version 4.1.1 in win32.
```

**Comment by axiom41** [comment]
Hello.
I tried myself and yes, the problem is reproduced also on my side  (Windows OS as well), but also with any of the "Force Tags" in a test suite.
As a simple workaround, after deleting a tag,  instead of pressing "ENTER", use the "Tab" key and "Force Tags" are correctly handled.
Here is my RIDE log:

20210322 11:15:27.951 [INFO]: Found Robot Framework version 3.2.2 from C:\Python\Python37\Lib\site-packages\robot.
20210322 11:15:27.951 [INFO]: Started RIDE 1.7.4.2 using python version 3.7.9 with wx version 4.0.7.post2 in win32.

**Comment by HelioGuilherme66** [comment]
@princeaok @axiom41 Could you check if installing the current beta 2.0 would have it fixed?
`pip install -U --pre robotframework-ride`

You can also try the current development version (just see the README doc)
(if it upgrades wxPython to 4.1.1, then it is better to later downgrade it, because that Beta was released before it)

**Comment by axiom41** [comment]
Hello.
Tried with v. 2.0b1 and it works. Btw, when do you thing we can have a stable 2.x release?
This is the updated log:
```
20210322 19:21:33.079 [INFO]: Found Robot Framework version 3.2.2 from C:\Python\Python37\lib\site-packages\robot.
20210322 19:21:33.080 [INFO]: Started RIDE v2.0b1 using python version 3.7.9 with wx version 4.1.1 in win32.
```

**Comment by HelioGuilherme66** [comment]
Closing because it is fixed on current 2.0 Beta 1.

(not possible to make a schedule release for 2.0, keep watching for incorporation of the pending Pull Requests)
